openedx/core/djangoapps/content/learning_sequences/api/processors/schedule.py

```diff
@@ -58,13 +66,20 @@ def inaccessible_sequences(self, full_course_outline):
         Sequences are inaccessible, regardless of the individual Sequence start
         dates.
         """
+        if self._is_beta_tester and self._days_early_for_beta is not None:
```

Comment:
`self._days_early_for_beta` does not have the correct value. Use the one from full_course_outline instead.